tests/src/actionContainers/SwiftActionContainerTests.scala

```diff
@@ -196,7 +198,29 @@ class SwiftActionContainerTests extends BasicActionRunnerTests with WskActorSyst
         })
     }
 
-        it should "properly use KituraNet and Dispatch" in {
+    it should "support pre-compiled binary in a zip file" in {
```

Comment:
It would be nice to have a test that has a zip that uses libraries that aren't already included in the Swift runtime.